### PR TITLE
Add bug report section

### PR DESCRIPTION
**Pull request by kasparm**
### Purpose
This PR adds the bug report section.
### Testing
Run tox locally and make sure addition works and the content is informative.
### Time estimate
10 min

### CODE REVIEW

File: source/dev/contribution_guide.rst [hunk 1]
```diff
@@ -29,6 +30,14 @@ How to Contribute
 #. Document your code (see the `Document your Code`_ section)
 #. Make a pull request (see our :doc:`../user/git_guide` page)
 
+Bug Report
```
Comment (jon-hagg) [inline source/dev/contribution_guide.rst:33]: Seems like this section is almost the same as for feature requests. Would it make sense to combine them? E.g. we could merge the bullet points into `Create an issue` and make this section generic, maybe `navigate to the repo, click on **Issues**.. select the appropriate template.. etc`

Comment (kasparm) [reply source/dev/contribution_guide.rst]: Thank you for your feedback. That makes sense. Originally I was just going about adding the section per the issues.
I made some modifications. Let me know if you like them.

File: source/dev/contribution_guide.rst [hunk 2]
```diff
@@ -20,20 +20,29 @@ How to Contribute
...
+++++++++++++++
+Thank you for reporting your bug or requesting a feature. Navigate to the repository you have identified
+as the most likely source of your bug or where you think your feature may belong to.
+
+If you are not sure if your bug is really a bug, but rather stems from you individual setup or installation,
+then use the `#questions <https://besciencescommunity.slack.com/archives/C01CBTRQQF9>`_ slack channel
+to discuss your issue. If you want to discuss your feature request first with the team, feel free to connect with us
+via the `#request_software <https://besciencescommunity.slack.com/archives/C01D594JXK3>`_ slack channel.
+
+If your are clear about your bug or feature request, choose the **Issues** tab on the repository and
```
Comment (jon-hagg) [inline source/dev/contribution_guide.rst:41]: your -> you

File: source/dev/contribution_guide.rst (hunk 1 above)
Comment (jon-hagg) [reply source/dev/contribution_guide.rst]: Thanks, looks good!

**Review by jon-hagg — APPROVED** [review]

**Pull request merged**